requirements-gui.txt

```diff
@@ -1,2 +1,7 @@
 pyqtgraph>=0.9.10
+
+# NOTE: What is this?
```

Comment:
orangeqt as a separate library. If I recall correctly, it is only used by parallel coordinates widget, the rest was already ported to pyqtgraph.